src/selectors/balances.js

```diff
@@ -44,63 +50,38 @@ import {
   activeAccountIdSelector,
   activeAccountSelector,
 } from './selectors';
-import { availableStakeSelector } from './paymentNetwork';
 
 
-export const accountBalancesSelector = createSelector(
+export const accountEthereumWalletBalancesSelector = createSelector(
   balancesSelector,
   activeAccountIdSelector,
-  (balances, activeAccountId) => {
+  (balances, activeAccountId): Balances => {
     if (!activeAccountId) return {};
-    return balances[activeAccountId] || {};
+    return balances?.[activeAccountId]?.[CHAIN.ETHEREUM] ?.[ASSET_CATEGORY.WALLET] || {};
```

Comment:
Is that space supposed to be there?

```suggestion
    return balances?.[activeAccountId]?.[CHAIN.ETHEREUM]?.[ASSET_CATEGORY.WALLET] || {};
```